cordova-plugin-medic/www/jasmine-jsreporter.js

```diff
@@ -1,214 +0,0 @@
-/*
-  This file is part of the Jasmine JSReporter project from Ivan De Marino.
-
-  Copyright (C) 2011 Ivan De Marino (aka detro, aka detronizator), http://blog.ivandemarino.me, ivan.de.marino@gmail.com
```

Comment:
This code isn't used in any one of `cordova-mobile-spec`, `cordova-plugin-test-framework`, or `cordova-medic`, so I presumed it to be dead. @dmitriy-barkalov, do you know what purpose that code serves?